deployment/ansible/scripts/topology.py

```diff
@@ -40,51 +45,132 @@ def hosts_to_peers(lines):
 def read_inventories(path):
     with open(path, "r") as f:
         lines = f.readlines()
-    key = lines[0].strip()
-    inventories = {key: []}
-    for _line in lines[1:]:
+    inventories = defaultdict(list)
+    for _line in lines:
         line = _line.strip()
         if not line.startswith("["):
             inventories[key].append(line)
         else:
             key = line
-            inventories[key] = []
-    print(inventories)
-    peers = hosts_to_peers(inventories["[nodes]"])
-    return peers
+    return inventories
 
 
-def barbell_topology(_inventories):
-    inventories = deepcopy(_inventories)
-    random.shuffle(inventories)
-    n = len(inventories)
-    topology = {peer: [inventories[(i + 1) % n]]
-                for i, peer in enumerate(inventories)}
-    return topology
+def barbell_topology(_peers):
+    peers = deepcopy(_peers)
+    random.seed(5566)
+    random.shuffle(peers)
+    n = len(peers)
+    topology = [{
+        'peer': peer,
+        'connect_to': [peers[(i + 1) % n]],
```

Comment:
Does this mean every peer is going to `addpeer` another peer? It will form a ring topo